src/composables/accounts.ts

```diff
@@ -87,15 +90,26 @@ const accounts = computed((): IAccount[] => {
   }
 
   const idxList = Object.fromEntries(PROTOCOL_LIST.map(((protocol) => [protocol, 0])));
+  let airGapIdx = 0;
 
   return accountsRaw.value
     .map((account, globalIdx) => {
+      if (account.type === ACCOUNT_AIR_GAP_WALLET) {
+        const airGapAccount = account as IAirgapAccountRaw;
+        const idx = airGapIdx;
+        airGapIdx += 1;
+
+        return {
+          globalIdx,
+          idx,
+          publicKey: Buffer.from(decode(airGapAccount.address as Encoded.AccountAddress)),
+          ...airGapAccount,
+        } as unknown as IAccount;
+      }
```

Comment:
Considering that we might have more account types in the future I suggest to make smal refactoring:
1. Instead of having `getHdWalletAccountFromMnemonicSeed` for each adapter I think we could have something more generic, like for example: `getAccountFromRawAccount` (or something similar). We would pass whole raw account and IDX.
2. The adapter would take care of checking of what kind of account it is and pick right logic to return correct data.

What do you think?